docker/maistra-builder_1.2.Dockerfile

```diff
@@ -1,10 +1,11 @@
 FROM fedora:31
 
 # Versions
-
+ENV GO_VERSION="1.14.4"
```

Comment:
Istio 1.6 integration tests require golang 1.14 (actually it's only one test that does, but well...)

Reply:
What about upgrading the base image to Fedora 32? It has go 1.14.

Reply:
Good point, but if possible I'd like to do that in a follow-up. I finally got this working and don't want to get it stuck on upgrade issues. As soon as it's rolled out I can look at upgrading to f32

Reply:
Fair enough